js/server/modules/@arangodb/foxx/manager.js

```diff
@@ -739,25 +712,34 @@ function _uninstall (mount, options = {}) {
     }
   }
   const collection = utils.getStorage();
-  const serviceDefinition = db._query(aql`
+  db._query(aql`
     FOR service IN ${collection}
     FILTER service.mount == ${mount}
     REMOVE service IN ${collection}
-    RETURN OLD
-  `).next();
-  if (serviceDefinition) {
-    const checksumRefs = db._query(aql`
-      FOR service IN ${collection}
-      FILTER service.checksum == ${serviceDefinition.checksum}
-      RETURN 1
-    `).toArray();
-    const bundleCollection = utils.getBundleStorage();
-    if (!checksumRefs.length && bundleCollection.exists(serviceDefinition.checksum)) {
-      bundleCollection.remove(serviceDefinition.checksum);
```

Comment:
Orphaned bundles should be cleaned up by the self-healing, so let's not worry about those here.